docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,27 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting and Getting the Server Icon URI
+
+To retrieve the server icon URI from the client, the server must set the URI in advance.
+If you try to read the server icon URI before it is set, the result is a null value.
+To retrieve the server icon URI from the client, follow these steps:
+
+1. Set the server icon on the server side:
```

Comment:
This are not really parallel. I mean, the client can read the iconURI  before server sets it, however in such case it will just be null. This order is important for app developers - at least this is what i think.

Reply:
@mijin85cho 
I agree with you, that the setting and getting are not sequential steps.  
According to my understanding setting and getting can happen simultaneously also.
However, the ideal way of retrieving server icon URI is to set the server icon URI and then reading it on the client side and will not return any null value. That is the reason I wrote, "To retrieve the server icon URI from the client, follow these steps:” 

@m-michalski2
Thank you very much for the clarification.

@mijin85cho and @m-michalski2
Can we rephrase as:
To set and get the server icon URI from the client, follow these steps:

Please verify and correct me if I am wrong.

Reply:
I propose we rewrite this entire paragraph this way:

Media Controller server can update its icon URI address, which can than be accessed from the           
client application. The default value of the server icon URI attribute is null. If you want to         
know how to use the icon URI attribute in your media controller application follow these steps:
...